doc/manual_experimental.md

```diff
@@ -2207,10 +2207,10 @@ NimPrinter().printConstRef(message, val)
 
 ```
 
-constructor pragma
+Constructor pragma
 ==================
 
-The `constructor` pragmas has two ways of being used: in conjunction with `importcpp` to import a C++ constructor and as a way to declare constructors that works similarly to `virtual`. 
+The `constructor` pragma has two ways of being used: in conjunction with `importcpp` to import a C++ constructor and as a way to declare constructors that work similarly to `virtual`. 
```

Comment:
Is it "a way that works" or "constructors that work"?

Reply:
I think you are right. ChatGPT suggests to use "The constructor pragma can be used in two ways: in conjunction with importcpp to import a C++ constructor, and to declare constructors that operate similarly to virtual."